.circleci/config.yml

```diff
@@ -272,6 +340,8 @@ executors:
     shell: /run/current-system/sw/bin/bash -leo pipefail
     working_directory: ~/expo
     environment:
+      # Set project directory as environment variable
+      PROJECT_DIRECTORY: /root/expo
```

Comment:
I think you don't have to define your own env var. Have you tried to use `CIRCLE_WORKING_DIRECTORY` instead?
https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables